bin/antigen.zsh

```diff
@@ -364,6 +375,8 @@ antigen () {
       branch="${url#*|}"
     fi
     install_or_update=true
+    # Update remote if needed.
+    -antigen-update-remote $clone_dir $url
```

Comment:
In the context you mention it doesn't make sense, you're right, I was thinking on current prezto users which will be moved to the community fork.

Now I see it'll create more confusion, with the `prezto repository url` config will be enough.